src/table/cells/renderer.js

```diff
@@ -6,9 +6,11 @@ import { withSelections } from './selections';
 
 // column will be needed in the future when column styling is implemented
 export function getCellRenderer(column, selectionsEnabled) {
+  // the column in the analysis mode
   if (selectionsEnabled) {
     return withSelections(TableCell);
   }
 
+  // the old column in the edit mode
```

Comment:
old column? We shouldn't reference edit/analysis mode yet, since it will be a while since the table is used in sense.